Source/JavaScriptCore/runtime/ArrayBuffer.cpp

```diff
@@ -212,6 +212,11 @@ Ref<ArrayBuffer> ArrayBuffer::create(ArrayBufferContents&& contents)
     return adoptRef(*new ArrayBuffer(WTFMove(contents)));
 }
 
+Ref<ArrayBuffer> ArrayBuffer::create(const Vector<uint8_t> vector)
+{
+    return ArrayBuffer::create(vector.data(), vector.size());
+}
```

Comment:
I cannot find the use of this function in this patch. If it is not used, then remove it.

Reply:
It's used now!